torch/_dynamo/__init__.py

```diff
@@ -80,10 +80,15 @@
 
 def reset() -> None:
     """Clear all compile caches and restore initial state"""
+    # TODO: https://github.com/pytorch/pytorch/issues/139200
```

Comment:
while you're touching these functions, can you add doc block on what they actually reset?